addon/collapse-key.js

```diff
@@ -5,11 +5,15 @@ export default function(property) {
     return [property];
   }
 
-  let atEachIndex = property.indexOf('.@each');
-  if (atEachIndex !== -1) {
-    return [property.slice(0, atEachIndex)];
-  } else if (property.slice(-2) === '[]') {
-    return [property.slice(0, -3)];
+  let atEachIndex = property.indexOf('@each.');
+  if (atEachIndex === -1) {
+    atEachIndex = property.indexOf('[]');
+  }
+
+  if (atEachIndex === 0) {
+    return [''];
```

Comment:
Hrrm, I can't get this to work. With this change, `collapsedKeys()` returns an empty array, then `bundledKeys` in `createArgs()` is an empty array, and `getValue()` never gets called, so there's no logic anywhere that sets the value at that key to `context`. What am I missing?